app/src/main/java/com/globant/scriptsapadea/ui/activities/ScreenPlayActivity.java

```diff
@@ -4,21 +4,20 @@
 import android.content.Context;
 import android.content.Intent;
 import android.os.Bundle;
+import android.support.v4.app.Fragment;
 import android.view.View;
 
 import com.globant.scriptsapadea.R;
-import com.globant.scriptsapadea.ui.fragments.PictureFragment;
+import com.globant.scriptsapadea.interfaces.OnNavigateToFragmentListener;
 import com.globant.scriptsapadea.ui.fragments.ScreenPlayFragment;
-import com.globant.scriptsapadea.ui.fragments.TakePictureFragment;
 
 import roboguice.inject.ContentView;
 
 @ContentView(R.layout.activity_screen)
-public class ScreenPlayActivity extends BaseActivity implements ScreenPlayFragment.OnScreenplayChangeFragmentListener,PictureFragment.OnSetPictureFragmentImageListener  {
+public class ScreenPlayActivity extends BaseActivity implements OnNavigateToFragmentListener {
```

Comment:
I needed to create similar interface to all fragments, I think is better to use a generic interface thant creating interface for each fragment, because the behavior is the same for all fragments, but if you said that I need to change it, I am going to change the interface